README.md

```diff
@@ -1,139 +1,466 @@
-This means that most files will start with `@use 00-config/functions/gesso as *`. This allows you to use the
-design token accessor functions without an additional namespace. Other functions and mixins can be used similarly. Note that to avoid namespace
-collisions, only Gesso-related variables, mixins, and functions should be used with `*`.
+Sass can be compiled as part of the global styles.css file or to individual CSS
+files for use in a Drupal library.
 
-All Sass files that are compiled to individual CSS files must have a unique name,
-even if they are in different directories.
+`@use` is used to import Sass variables, mixins, and/or functions into
+individual SCSS files. [`@import` is discouraged by the Sass team and will
+eventually be phased out.](https://sass-lang.com/documentation/at-rules/import).
+This means that most files will start with
+`@use 00-config/functions/gesso as *`. This allows you to use the design token
+accessor functions without an additional namespace. Other functions and mixins
+can be used similarly. Note that to avoid namespace collisions, only
+Gesso-related variables, mixins, and functions should be used with `*`.
 
-### Global Styles
-Prefix the name of your Sass file with `_`, e.g. `_card.scss`. Add it to the appropriate
-aggregate file (i.e. `_components.scss`).
+All Sass files that are compiled to individual CSS files must have a unique
+filename, even if they are in different directories.
 
-### Individual Component/Library Styles
-DO NOT prefix the name of your Sass file with `_`, e.g. `menu.scss`. Import the config
-and global aggregate files. Import your SCSS file at the top of your Storybook file.
-See `menu.stories.jsx` for an example. Don't forget to add it to the `gesso.libraries.yml`
-file as well.
+### Global styles
+
+Prefix the name of your Sass file with `_`, e.g. `_card.scss`. Add it to the
+appropriate aggregate file (i.e. `_components.scss`).
+
+### Individual component/library styles
+
+DO NOT prefix the name of your Sass file with `_`, e.g. `menu.scss`. Import the
+config and global aggregate files. Import your SCSS file at the top of your
+Storybook file. See `menu.stories.jsx` for an example. Don’t forget to add it to
+the `gesso.libraries.yml` file as well.
 
 ### Linting
-Stylelint and Prettier are used to lint CSS and SCSS files.
+
+Stylelint and Prettier are used to lint CSS and SCSS files. Warnings will
+break the build, so if you have a valid reason to break Stylelint rules you can
+have it ignore code in two ways:
+
+1.  Add `// stylelint-disable-next-line` to the line just before where the
+    Stylelint warning is triggered.
+
+2.  To ignore several lines, add `// stylelint-disable` before the code in
+    question and add `// stylelint-enable` afterwards.
+
+The Stylelint rules can be changed in the `.stylelintrc.yml` file. By default,
```

Comment:
Change from our previous documentation, but I wonder if we should include something here that you should also leave a comment as to what the valid reason was that required you to disable stylelint (or eslint etc.) That both forces you to think about whether you really have a valid reason and can alert your code reviewer/future developers as to whether that can/should be changed. (I've had a code reviewer catch that my reason for disabling wasn't actually correct and I didn't need to disable the linter; I needed to fix the bug three lines up.)

Reply:
Oooh, I like this! Adding more info now.